### PR TITLE
New keyboard shortcut in product page

### PR DESCRIPTION
**Pull request by AntoineMille**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Please be specific when describing the PR. <br/> Every detail helps: versions, browser/server configuration, specific module/theme, etc. |
| Type? | new feature |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1394 |
| How to test? | Go to the product page in the BO and test CTRL+V & CTRL+O |

### CODE REVIEW

**Comment by dSkrbic** [comment]
Test Q&A **OK**

**Comment by aleeks** [comment]
We need to rebase @AntoineMille See with you this afternoon!

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -933,6 +933,21 @@ var form = (function() {
...
+      /** create keyboard event for save & go preview */
+      jwerty.key('ctrl+V', function(e) {
+          e.preventDefault();
+          send($('.product-footer .preview').attr('data-redirect'), $('.product-footer .preview').attr('target'));
+      });
+
+      /** create keyboard event for save & active or desactive product*/
+      jwerty.key('ctrl+O', function(e) {
+        e.preventDefault();
+        if ($('#form_step1_active').prop('checked') == true)
```
Comment (aleeks) [inline admin-dev/themes/default/js/bundle/product/form.js]: Why not just : 
`var step1Checkbox = $('#form_step1_active');`
`step1Checkbox.prop('checked', !step1Checkbox.is(':checked'));`

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 2]
```diff
@@ -933,6 +933,21 @@ var form = (function() {
         send($('.product-footer .go-catalog').attr('data-redirect'));
       });
 
+      /** create keyboard event for save & go preview */
+      jwerty.key('ctrl+V', function(e) {
+          e.preventDefault();
+          send($('.product-footer .preview').attr('data-redirect'), $('.product-footer .preview').attr('target'));
```
Comment (aleeks) [inline admin-dev/themes/default/js/bundle/product/form.js]: Why not ? : (ping other, i'm not sure)
`var productFooter = $('.product-footer .preview');`
`send(productFooter.attr('data-redirect'), productFooter.attr('target'));`

Comment (AntoineMille) [inline admin-dev/themes/default/js/bundle/product/form.js]: I'm agree with you to use var. Sorry for this.

**Comment by aleeks** [comment]
Thank you for your changes, i'm waiting test :) !

**Pull request merged**